jazzmin/__init__.py

```diff
@@ -1,6 +1 @@
-import django
-
-version = "2.6.1"
-
-if django.VERSION < (3, 2):
-    default_app_config = "jazzmin.apps.JazzminConfig"
+version = "3.0.0"
```

Comment:
Versions no longer tracked here, we can set this to 0.0.0

Reply:
Being fixed in this PR to remove the hardcoding